great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -1747,10 +1747,26 @@ def _pandas_map_condition_index(
 
     df = df[boolean_mapped_unexpected_values]
 
-    if result_format["result_format"] == "COMPLETE":
-        return list(df.index)
+    if "unexpected_index_columns" in result_format:
 
-    return list(df.index[: result_format["partial_unexpected_count"]])
```

Comment:
Where is the partial_unexpected_count handled if this is removed?

Reply:
Thank you for this question @anthonyburdi. The partial counts are handled at the  

[`_format_map_output()`](https://github.com/great-expectations/great_expectations/blob/6c1fead10e3e3c70e78c0c9f1150d9516b169356/great_expectations/expectations/expectation.py#L2912) - level, which progressively adds more output as the `result_format` settings become more verbose.  That work makes the filtering at the Metric level redundant.